client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.tsx

```diff
@@ -201,6 +202,21 @@ function LineItemWrapper( {
 		if ( isJetpack || isAkismet ) {
 			return true;
 		}
+
+		try {
+			const { variationName } = dangerouslyGetExperimentAssignment( 'calypso_plans_2yr_toggle' );
+
+			if ( variationName === 'toggle_and_checkout' ) {
+				return true;
+			}
+		} catch {
+			// The error is intentionally ignore here. For this particular experiment,
+			// the experience should start from the 2023 version of plans page to the checkout.
+			// Thus, for any other flow that leads to the checkout, they shouldn't be affected.
+			// That also means chances are that we can't load or preload the experiment earlier
+			// so that `dangerouslyGetExperimentAssignment` doesn't throw.
+		}
```

Comment:
@southp, there was no error being thrown and the explat-client function was logging directly the error as reported in p1680892941597569-slack-C02DQP0FP. We're disabling the error logging for the purpose of this experiment in https://github.com/Automattic/wp-calypso/pull/75442